### ISSUE TITLE
Add a section for custom rolls

### ISSUE THREAD

**Issue by scottbenton** [scottbenton/Iron-Journal#272]

**Comment by annabranco** [comment]
Is this to allow players to physically rolls dice, for example, and input their results?

**Comment by scottbenton** [comment]
No, but that's also an interesting idea! This idea is more so allowing people to roll dice without having to select a stat or an oracle first. For example, in the game I'm in, our GM wanted us to recently roll 1d100 - not against any table, just by itself. Someone else in the discord mentioned needing to roll a d66 for example, so I think being flexible about the type & amount of dice would be useful.

**Comment by tslays** [comment]
+1 for a physical dice roll with manual input. Foundry has it now but they got the auto focus wrong. 

e.g. I am in manual roll mode (toggle via settings), I click on roll, focus is already in the input field and I just need to type, focus jumps automatically after entering a 4 as action roll, focus on 1st challenge die, I type 2, focus on 2nd challenge die, I type 0 (for 10) and it shows me the result.
